src/Input/Keybindings.re

```diff
@@ -84,9 +91,49 @@ module Internal = {
     };
 };
 
-let empty = [];
+module Input =
+  EditorInput.Make({
+    type context = Hashtbl.t(string, bool);
+    type payload = string;
```

Comment:
It's also very unclear for me what `payload` is. What is an example of a payload value in this context?

Reply:
Ah, `payload` is a poor name. I refactored it to `command` in this PR: https://github.com/onivim/editor-input/pull/9

`action` might also be a reasonable name. It's an object you associate with a key binding, that, when the key binding is engaged, it provides with an `Execute` effect.

For example, if you have a binding `j`, like:
```
let alwaysEnabled = _context => true;
let bindings = 
Input.empy |> Input.addBinding("j", alwaysEnabled, "jPressed");

let (_bindings, effects)  = Input.keyDown(~context, ~jKey, bindings);
// effects == [Execute("jPressed")]
```

Reply:
Ah, yeah `command` makes sense I think!